src/components/atoms/Option.vue

```diff
@@ -2,37 +2,40 @@
   <div id="option">
     <div class="checkbox-and-label" @click="toggleCheck">
       <div class="checkbox-circle">
-        <div v-if="checked" class="checked"></div>
+        <div v-if="name === selectedOption" class="checked"></div>
       </div>
-      <span>{{ name }}</span>
+      <span class="checkbox-name">{{ name }}</span>
     </div>
   </div>
 </template>
 
 
 <script>
-import {COLORS, FONT_SIZES} from "../../../public/Consts";
+import { COLORS, FONT_SIZES } from "../../../public/Consts";
 export default {
   // eslint-disable-next-line vue/multi-word-component-names
   name: "Option",
   data() {
     return {
-      checked: false,
       COLORS: COLORS,
       FONT_SIZES: FONT_SIZES,
+      isSelected: false,
     }
   },
-  props:
-      {
-        name:
-            {
-              type: String,
-              required: true,
-            },
-      },
+  props: {
+    name: {
+      type: String,
+      required: true,
+    },
+    selectedOption: {
+      type: String,
+      default: null,
+    },
+  },
   methods: {
     toggleCheck() {
-      this.checked = !this.checked;
+      console.log('Option clicked: ', this.name);
+      this.$emit('selected', this.name);
```

Comment:
change to this.emitter.emit if necessary, but if so why is there another emit in FilterOptions?

Reply:
My approach was the Option has to emit an emit on its own, so that FilterOptions can ensure there's only one option selected at a single time.